src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -1103,7 +1113,7 @@ private TransformEngineChain createTransformEngineChain()
 
         // TODO RTCP termination should end up here.
 
-        RemoteBitrateEstimator
+        RemoteBitrateEstimatorWrapper
```

Comment:
again, can we get away with just the `RemoteBitrateEstimator` type?

Reply:
The `RemoteBitrateEstimator` is not a `TransformEngine` (nor should it).